<!-- robot is stationary -->
<launch>
    <node name="ArucoDetect" pkg="aruco_detect" type="aruco_detect" output="log"/>
    <node name="ArucoPub" pkg="stalkerbot" type="Aruco_Pub.py" output="log"/>
    <node name="fiducial_filter" pkg="stalkerbot"= type="fiducial_filter.py" output="log"/>
    <node name="location_publisher" pkg="stalkerbot" type="location_pub.py" output="screen"/>
    <node name="teleop" pkg="turtlebot3_teleop" type="turtlebot3_teleop_key.launch" output="log"/>
    <node name="republish" type="republish" pkg="image_transport" output="screen" args="compressed in:=raspicam_node/image raw out:=/camera"/>
</launch>